docs/using-stacks.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I believe @suskin has named this `stacks-guide.md`

Reply:
fixed